presto-main/src/main/java/com/facebook/presto/event/QueryMonitor.java

```diff
@@ -410,15 +412,15 @@ private QueryStatistics createQueryStatistics(QueryInfo queryInfo)
                 queryStats.getRuntimeStats());
     }
 
-    private QueryContext createQueryContext(SessionRepresentation session, Optional<ResourceGroupId> resourceGroup)
+    private QueryContext createQueryContext(SessionRepresentation session, Optional<ResourceGroupId> resourceGroup, Set<String> extraClientTags)
```

Comment:
Can we create overloaded method with default value for extraClientTags as `ImmutableSet.of()`. This will avoid changing all the other places where we are simply passing empty maps